apps/sumo-aws-apps/vpc-flow-logs/vpcflowlogs.template.yaml

```diff
@@ -123,13 +126,12 @@ Parameters:
     ConstraintDescription: "Sumo Logic Organization Id can not be empty."
   RemoveSumoResourcesOnDeleteStack:
     AllowedValues:
-      - true
-      - false
-    Default: true
+      - 'True'
```

Comment:
actually we should test with different values of true and false for RemoveSumoResourcesOnDeleteStack parameter and check whether it deletes resources or not in sumo. My hunch is this will fail because in helper function we treat it as boolean https://github.com/SumoLogic/sumologic-aws-lambda/blob/main/sumologic-app-utils/src/sumoresource.py#L135